voctogui/lib/audioleveldisplay.py

```diff
@@ -1,17 +1,29 @@
 import math
 import cairo
 
-from gi.repository import Gtk, GLib
+from lib.config import Config
+from gi.repository import Gtk, GLib, Gst
 
 
 class AudioLevelDisplay(Gtk.DrawingArea):
     """Displays a Level-Meter of another VideoDisplay into a GtkWidget"""
     __gtype_name__ = 'AudioLevelDisplay'
 
     def __init__(self):
-        self.levelrms = []
-        self.levelpeak = []
-        self.leveldecay = []
+        self.audiostreams = int(Config.get('mix', 'audiostreams'))
+        meters = Config.get('mainvideo', 'vumeter')
+        if (Config.has_option('mainvideo', 'vumeter')) \
+                and (meters != 'all') \
+                and (int(meters) < self.audiostreams):
+            self.audiostreams = int(meters)
```

Comment:
from the name `self.audiostreams` it is not clear, what type this actually is. I'd prefere `num_audiostreams_`.
Did you try PyCharm? You can easily refactor thing like this.